README.md

```diff
@@ -39,6 +39,10 @@ Full documentation is available in this project's web-based pattern library.
    ```
 2. Visit http://localhost:4001/ in a browser
 
+The documentation is also publicly available in https://??.hypothes.is/.
+
+It gets automatically updated every time a new version is released, but it can also be manually deployed using the [deploy workflow](https://github.com/hypothesis/frontend-shared/actions/workflows/deploy.yml)
```

Comment:
I don't think we need this instruction in the general `README`, which isn't targeted at folks doing development.

Reply:
ok!